docs/kits/Business Partner Kit/Documentation BPDM/page_use-cases.md

```diff
@@ -0,0 +1,2294 @@
+                        "shortName": null,
+                        "type": {
+                            "technicalKey": "LOCAL",
+                            "name": "The business partner name identifies a business partner in a given context, e.g. a country or region.",
+                            "url": ""
+                        },
+                        "language": {
+                            "technicalKey": "undefined",
+                            "name": "Undefined"
+                        }
+                    }
+                ],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-06-03T11:46:42.891266Z"
+            }
+        },
+        {
+            "score": 19.11017,
+            "legalEntity": {
+                "bpn": "BPNL000000045YT4",
+                "identifiers": [
+                    {
+                        "value": "DE291361833",
+                        "type": {
+                            "technicalKey": "EU_VAT_ID_DE",
+                            "name": "Umsatzsteuer-Identifikationsnummer",
+                            "url": "https://meta.cdq.com/European_value_added_tax_identifier_(Germany)"
+                        },
+                        "issuingBody": null,
+                        "status": null
+                    },
+                    {
+                        "value": "CX.POOL:62DE4C8F1E6B3B3852691C9E0",
+                        "type": {
+                            "technicalKey": "CX_POOL_ID",
+                            "name": "",
+                            "url": null
+                        },
+                        "issuingBody": null,
+                        "status": null
+                    }
+                ],
+                "names": [
+                    {
+                        "value": "ColorDruck Solutions GmbH",
+                        "shortName": null,
+                        "type": {
+                            "technicalKey": "LOCAL",
+                            "name": "The business partner name identifies a business partner in a given context, e.g. a country or region.",
+                            "url": ""
+                        },
+                        "language": {
+                            "technicalKey": "undefined",
+                            "name": "Undefined"
+                        }
+                    }
+                ],
+                "legalForm": null,
+                "status": null,
+                "profileClassifications": [],
+                "types": [],
+                "bankAccounts": [],
+                "roles": [],
+                "relations": [],
+                "currentness": "2022-08-10T10:33:35.402952Z"
+            }
+        }
+    ]
+}
+```
```

Comment:
Also here, shorten the return call, so the user can get the idea without too much information overload